staging/src/k8s.io/client-go/examples/create-update-delete-deployment/main.go

```diff
@@ -167,7 +167,6 @@ func prompt() {
 	fmt.Printf("-> Press Return key to continue.")
 	scanner := bufio.NewScanner(os.Stdin)
 	for scanner.Scan() {
-		break
```

Comment:
That definitely changes the behavior?

Reply:
Oops! My fault.
You are right. This should not be removed.
I changed the PR, PTAL. Thanks.